RecaptchaInterop.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'RecaptchaInterop'
-  s.version          = '100.0.0'
+  s.version          = '101.0.0'
```

Comment:
I _think_ this should be `100.1.0` since it's not a breaking change (purely additive) and based on the guidance in our [Versioning](https://github.com/google/interop-ios-for-google-sdks#versioning) strategy.

Reply:
I was in doubt if it should be `100.1.0` or `100.0.0`. The new version of the reCAPTCHA SDK should not be used with the old Interop but just with the new one. Basically what we want is that from the reCAPTCHA 18.0.0 to 18.6.0 you can use the old Interop and from 18.7.0 you need to use the new Interop and therefore the new reCAPTCHA SDK. Right now the `Package.swift` of the reCAPTCHA SDK is `"100.0.0"..<"101.0.0"` what would make it support `100.1.0` in my understanding. And Cocoapods is `~> 100.0.0` that I have the impression that will also accept `100.1.0` and this should not be supported. Is my understanding correct?

Reply:
Should be 101.0.0 since it's a breaking change to add a method to a protocol that clients will need to implement.